src/lib.rs

```diff
@@ -209,6 +209,7 @@ mod arena;
 pub mod back;
 mod block;
 pub mod front;
+pub(crate) mod keywords;
```

Comment:
`pub(crate)` doesn't make sense in this place. Just remove it, leave only `mod keywords`.